.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: required
+
+services:
+  - docker
+
+env:
+  SHORT_COMMIT=`echo $TRAVIS_COMMIT | cut -c1-7`
+  CURRENT_DATE=`date '+%Y%m%d%H%M%S'`
+  TAG="${CURRENT_DATE}_${SHORT_COMMIT}"
+  PATH=/snap/bin:$PATH
+
+before_install:
+  - sudo apt-get update && sudo apt-get install --only-upgrade openssl
+  - sudo pip install --user urllib3[secure] --upgrade
```

Comment:
Minor: consider freezing versions on package installs here for stability in the future

Reply:
This isn't for Parliament2 itself but for Travis CI to work, so I'll keep it as-is for now.